### DISCUSSION TITLE
-bash: cms/settings.py: Permission denied

### DISCUSSION THREAD

**Discussion by Wout01** [mediacms-io/mediacms#206] [Q&A] [answered]
Hey,

Does anyone here know how to access 'cms/settings.py' because when I run that command I get this:"-bash: cms/settings.py: Permission denied". Does anyone know how I solve this?

**Comment by mgogoulos** [comment]
cms/settings.py is a file with settings, so you are not supposed to run it. What are you trying to achieve?

**Comment by Wout01** [comment]
For some things to change

**Reply by mgogoulos** [reply]
it's not an executable, so don't run it but rather open it with an editor!

However I'd suggest you see what changes you want to make there, and keep them separate on a new file cms/local_settings.py

So for example if you want to edit the setting `DEFAULT_THEME = "light"` that is defined on cms/settings.py, make a new file cms/local_settings.py and set `DEFAULT_THEME = "dark"` there.

**Comment by Wout01** [comment]
Thanks

**Comment by Wout01** [comment, marked as answer]
I can't change anything at all, not even in other files like for example 'login.html' it says permission denied every time

**Reply by Dan1ell** [reply]
Well, what command are you using to change things?

**Reply by Wout01** [reply]
It seems I found the problem after a long search. For example, if I ran the command:"cms/local_settings.py" I got 'Permission denied' but if I put 'cat' in front of it it works. Could that be right?

**Reply by Dan1ell** [reply]
cat means concatenate. If you write cat > filename, it thinks you want to concatenate text to the end of that file. I don't think that's what you want.

You can open the file from the command line with the text editor nano:
`nano cms/local_settings.py`
I suggest you google nano before continuing.

And if you get permission denied, you can apply superuser rights:
`sudo nano cms/local_settings.py`
I suggest you google the sudo command before continuing.

Not sure if your code is on a laptop or on a server, but if it's on a laptop you could just use a regular editor like visual studio code. No need to use the command line in that case.

Good luck!

**Comment by Wout01** [comment]
Thanks!
